pallets/manual-migrations/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "pallet-manual-migrations"
```

Comment:
It's better to add moonbeam in the name of the pallet to be explicit ion the fact that's it's a moonbeam specific pallet, we did that with the orbiter pallet for example

Reply:
Renamed the pallet to `pallet-moonbeam-lazy-migrations`